### DISCUSSION TITLE
Make wezterm compliant with the XDG Base Directory specification

### DISCUSSION THREAD

**Discussion by ghost** [wezterm/wezterm#2061] [Ideas]
...meaning that the configuration file on Linux/Mac/BSD will be searched at `$XDG_CONFIG_HOME/wezterm/wezterm.lua` if the variable is set. If the variable is not set, default to `$HOME/.config/wezterm/wezterm.lua`. As far as I can tell, this is not implemented currently.

[XDG Base Directory specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html)

**Comment by wez** [comment]
That is how it works today:

https://github.com/wez/wezterm/blob/1ae72bccae7f381fd32d72173ab48653167c6bf9/config/src/lib.rs#L318-L323

**Reply by ghost** [reply]
Ohh great! it was not mentioned in the documentation on https://wezfurlong.org/wezterm/config/files.html. I tested it but i must have done something wrong. Thanks!

**Reply by wez** [reply]
I updated the docs to reflect this
